Server.js

```diff
@@ -363,12 +363,22 @@ Server.prototype = {
 			desiredCapabilities: desiredCapabilities,
 			requiredCapabilities: requiredCapabilities
 		}).then(function (response) {
-			// At least geckodriver 0.15.0 returns the response data in a 'value' property, whereas Selenium does not.
-			if (response.value && response.value.sessionId) {
-				response = response.value;
+			var responseData = null;
+			var sessionId = null;
+			// At least geckodriver 0.19.0 returns the response data in a 'value.capabilities'
+	        // property, whereas Selenium does not.
```

Comment:
Somehow my commit to fix the mixed use of tabs and spaces in this and the next few lines was lost with the recent push...